src/patternfly/components/Select/select.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="pf-c-select{{#if select--IsExpanded}} pf-m-expanded{{/if}}{{#if select--modifier}} {{select--modifier}}{{/if}}"
+<div class="pf-c-select{{#if select--IsExpanded}} pf-m-expanded{{/if}}{{#if select-menu-modifier}} {{select-menu-modifier}}{{/if}}"
```

Comment:
```suggestion
<div class="pf-c-select{{#if select--IsExpanded}} pf-m-expanded{{/if}}{{#if select--modifier}} {{select--modifier}}{{/if}}"
```

Reply:
Since this is the `select` component/hbs file, we want to be able to pass `select--modifier` to it to pass css classes.